upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -676,6 +714,17 @@ webhooks:
   sideEffects: None
   name: defaulting.webhook.provisioners.karpenter.sh
   rules:
+  - apiGroups:
+    - karpenter.k8s.aws
+    apiVersions:
+    - v1alpha1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - awsnodetemplates
```

Comment:
Are these CRDs managed by karpenter itself? I don't see that we are applying them to the cluster.